pkg/NAMESPACE

```diff
@@ -132,6 +134,9 @@ exportMethods("asc",
               "countDistinct",
               "sumDistinct")
 
+exportClasses("GroupedData")
+exportMethods("count", "agg", "avg")
```

Comment:
I think these methods are already exported by other classes like `RDD` or `Column` so we strictly don't need them here.